### PR TITLE
[docs](load) add data source docs

### PR DESCRIPTION
**Pull request by kaijchen**
## Versions 

- [x] dev
- [x] 3.0
- [x] 2.1
- [ ] 2.0

## Languages

- [x] Chinese
- [x] English

## Docs Checklist

- [ ] Checked by AI
- [ ] Test Cases Built

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-source/amazon-s3.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+Doris 提供两种方式从 AWS S3 导入文件：
+- 使用 S3 Load 将 S3 文件导入到 Doris 中，这是一个异步的导入方式。
```
Comment (liaoxin01) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-source/amazon-s3.md:28]: s3 load 和 tvf在这里加下链接的

File: sidebars.json [hunk 2]
```diff
@@ -157,6 +157,21 @@
                             "label": "Loading Data",
                             "items": [
                                 "data-operate/import/load-manual",
+                                {
+                                    "type": "category",
+                                    "label": "Data Source",
+                                    "items": [
+                                        "data-operate/import/data-source/hdfs",
```
Comment (liaoxin01) [inline sidebars.json:164]: hdfs放到对象存储的后面吧

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-source/minio.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    age                INT                   COMMENT "age"
+)
+DUPLICATE KEY(user_id)
+DISTRIBUTED BY HASH(user_id) BUCKETS 10;
+```
+
+### 第 3 步：使用 S3 Load 导入数据
+
+:::caution 注意
+如果您在本地网络中部署了 MinIO 并且未启用 TLS，则需要在 endpoint 字符串中明确添加 `http://`。
```
Comment (liaoxin01) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-source/minio.md:67]: 加个use_path_style的提示

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-source/minio.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+- `"s3.endpoint" = "http://localhost:9000"`
```
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-source/minio.md:69]:
```suggestion
- `"s3.endpoint" = "http://localhost:9000"`

S3 SDK 默认使用 virtual-hosted style 方式。但 MinIO 默认没开启 virtual-hosted style 方式的访问，此时我们可以添加 `use_path_style` 参数来强制使用 path style 方式。

- `"use_path_style" = "true"`
```

File: docs/data-operate/import/data-source/minio.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+DUPLICATE KEY(user_id)
+DISTRIBUTED BY HASH(user_id) BUCKETS 10;
+```
+
+### Step 3: Load data using S3 Load
+
+:::caution Caution
+If you deployed MinIO in a local network and did not have TLS enabled, you need to explicitly add `http://` in the endpoint string.
+
+- `"s3.endpoint" = "http://localhost:9000"`
```
Comment (kaijchen) [inline docs/data-operate/import/data-source/minio.md:69]:
```suggestion
- `"s3.endpoint" = "http://localhost:9000"`

The S3 SDK uses the virtual-hosted style by default. However, MinIO does not enable virtual-hosted style access by default. In this case, we can add the `use_path_style` parameter to force the use of the path style.

- `"use_path_style" = "true"`
```

File: docs/data-operate/import/data-source/minio.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+DUPLICATE KEY(user_id)
+DISTRIBUTED BY HASH(user_id) BUCKETS 10;
+```
+
+### Step 3: Load data using TVF
+
+:::caution Caution
+If you deployed MinIO in a local network and did not have TLS enabled, you need to explicitly add `http://` in the endpoint string.
+
+- `"s3.endpoint" = "http://localhost:9000"`
```
Comment (kaijchen) [inline docs/data-operate/import/data-source/minio.md:159]:
```suggestion
- `"s3.endpoint" = "http://localhost:9000"`


The S3 SDK uses the virtual-hosted style by default. However, MinIO does not enable virtual-hosted style access by default. In this case, we can add the `use_path_style` parameter to force the use of the path style.

- `"use_path_style" = "true"`
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-source/minio.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+DUPLICATE KEY(user_id)
+DISTRIBUTED BY HASH(user_id) BUCKETS 10;
+```
+
+### 第 3 步：使用 TVF 导入数据
+
+:::caution 注意
+如果您在本地网络中部署了 MinIO 并且未启用 TLS，则需要在 endpoint 字符串中明确添加 `http://`。
+
+- `"s3.endpoint" = "http://localhost:9000"`
```
Comment (kaijchen) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data-operate/import/data-source/minio.md:159]:
```suggestion
- `"s3.endpoint" = "http://localhost:9000"`

S3 SDK 默认使用 virtual-hosted style 方式。但 MinIO 默认没开启 virtual-hosted style 方式的访问，此时我们可以添加 `use_path_style` 参数来强制使用 path style 方式。

- `"use_path_style" = "true"`
```

**Comment by KassieZ** [comment]
Please fix deadlink, and request review @wangtianyi2004

**Comment by wangtianyi2004** [comment]
LGTM

**Pull request merged**